### PR TITLE
Added primary and secondary buttons

### PR DESCRIPTION
**Pull request by naijwu**
Added primary and secondary buttons with their default, hovered, focused, inactive states

Spent an hour trying to get storybook to work but realized the button component just fully didn't exist yet ;;

### CODE REVIEW

**Comment by naijwu** [comment]
Will change component name

File: src/components/button/Button.js [hunk 1]
```diff
@@ -1 +1,80 @@
...
+        font-weight: bold;
+
+        transition: all 0.13s linear; 
+        position: relative;
+        overflow: hidden;
+        cursor: pointer;
+        
+        position: relative;
+
+        ${p => p.active
```
Comment (ajo01) [inline src/components/button/Button.js:21]: I would suggest making separate methods for getting background, active, etc styles as this isn't the easiest to read. So for background styles something like:
`  const getBackgroundStyles = (p) => {
    if (p.variant === "primary") {
      return "background: linear-gradient(to bottom right, #0DEFE1, #78FF96); color: #2C2543; ";
    } else if (p.variant === "secondary") {
      return "background: #FFF;";
    } else {
      return "background: #BDBAC3; color: #8C898F; cursor: default;";
    }
  };`

with  ` ${(p) => getBackgroundStyles(p)}` inside the styled div.

Comment (naijwu) [reply src/components/button/Button.js:21]: that makes a whole lot of sense! Thank you

**Review by ajo01 — APPROVED** [review]
looks good!

**Comment by naijwu** [comment]
@phoenixthefirebird ok to merge?

**Review by phoenixthefirebird — APPROVED** [review]
Great work!

**Pull request merged**